tensornetwork/backends/jax/jitted_functions.py

```diff
@@ -202,32 +204,10 @@ def _generate_arnoldi_factorization(jax: types.ModuleType) -> Callable:
 
   """
 
-  @jax.jit
-  def modified_gram_schmidt_step_arnoldi(j, vals):
-    """
-    Single step of a modified gram-schmidt orthogonalization.
-    Args:
-      j: Integer value denoting the vector to be orthogonalized.
-      vals: A list of variables:
-        `vector`: The current vector to be orthogonalized
-        to all previous ones
-        `krylov_vectors`: jax.array of collected krylov vectors
-        `n`: integer denoting the column-position of the overlap
-          <`krylov_vector`|`vector`> within `H`.
-    Returns:
-      updated vals.
 
-    """
-    vector, krylov_vectors, n, H = vals
-    v = krylov_vectors[j, :]
-    h = jax.numpy.vdot(v, vector)
-    H = jax.ops.index_update(H, jax.ops.index[j, n], h)
-    vector = vector - h * jax.numpy.reshape(v, vector.shape)
-    return [vector, krylov_vectors, n, H]
```

Comment:
yes, that has to go because I had to move the definition of `_gs_step` inside the scope of `precision` (I couldn't find another way of making this work with `jax.lax.scan`).